base/pkg/entry.jl

```diff
@@ -50,19 +50,20 @@ function add(pkg::AbstractString, vers::VersionSet)
     outdated = :maybe
     @sync begin
         @async if !edit(Reqs.add,pkg,vers)
-            ispath(pkg) || throw(PkgError("unknown package $pkg"))
```

Comment:
I disagree. It is a bad practice to clutter user interface with unwanted information. It creates confusion and sense of instability.